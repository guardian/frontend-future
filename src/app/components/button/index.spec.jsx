// @flow

jest.mock('./style.scss', () => ({
    button: {
        backgroundColor: 'hotpink',
    },
}));

const Button = require('.');
const style = require('./style.scss');

console.log(style.button);

describe('Button', () => {
    it('generates the correct object', () => {
        expect(<Button>test button</Button>).toMatchSnapshot();
    });
});
