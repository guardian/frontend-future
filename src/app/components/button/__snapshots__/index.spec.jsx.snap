// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button generates the correct object 1`] = `
VNode {
  "attributes": undefined,
  "children": Array [
    "test button",
  ],
  "key": undefined,
  "nodeName": Object {
    "default": [Function],
  },
}
`;
